src/components/Select/ComboboxMulti/ComboboxMulti.tsx

```diff
@@ -117,6 +118,7 @@ export function ComboboxMulti<S>(props: Props<S>) {
 ComboboxMulti.defaultProps = {
     showClearButton: false,
```

Comment:
Just noticed that this default prop doesn't have any matching one in propType object. It extends `CommonProps` from SelectBase but there's no `showClearButton` as well. I wonder if it was required for this component to be extended with Clear button in the first place? If it's so we should extend ComboboxMulti props with `CommonPropsWithClear` and if it's not true remove `showClearButton` from defaultProps.